src/aspect.ts

```diff
@@ -281,7 +281,7 @@ function getDispatcherObject(target: Targetable, methodName: string): Dispatcher
 function getJoinPointDispatcher<F extends GenericFunction<T>, T>(joinPoint: F): F {
 
 	function dispatcher(this: Function, ...args: any[]): T {
-		const { before, after, joinPoint } = dispatchAdviceMap.get(dispatcher);
+		const { before, after, joinPoint } = dispatchAdviceMap.get(dispatcher)!;
```

Comment:
I have added comments to document the use of not null assertion is appropriate because we can't logically have `undefined` values from the maps.